web/web_ui/src/main/webapp/WEB-INF/web.xml

```diff
@@ -24,7 +24,7 @@
 
   <context-param>
     <param-name>javax.faces.PROJECT_STAGE</param-name>
-    <param-value>Production</param-value>
+    <param-value>Development</param-value>
```

Comment:
@Zakaria-Kofiro Can you explain the reason for this change?

Reply:
Oh, this is a change Kevin suggested where setting this from Production to Development enables error messages logs that wouldn't normally appear if set to Production, to help with debugging. Basically enables additional logging. 

https://stackoverflow.com/questions/19193912/uses-of-javax-faces-project-stage